### ISSUE TITLE
nightly docker image build

### ISSUE THREAD

**Issue by mhrimaz** [eclipse-basyx/basyx-java-server-sdk#41]
Is there any plan to have nightly(latest) docker images? The current docker images do not reflect the latest version of basyx.

For example the concept description image doesn't have pagination, though if you build and run locally, you would have pagination information in response. This is very confusing.

I built it locally in my own way. But, where can I find the information regarding building docker images locally? The [documentation ](https://wiki.eclipse.org/Talk:BaSyx_/_Documentation_/_Components_/_Docker) is for September 2022 which is not clear for me if it is updated and still relevant or not. Also, I got this error by following the maven command `mvn install -Pdocker` in the documentation:
`[WARNING] The requested profile "docker" could not be activated because it does not exist.`

**Comment by FrankSchnicke** [comment]
We're planning to provide releases roughly every two weeks, if something new was implemented. 

We merged https://github.com/eclipse-basyx/basyx-java-server-sdk/pull/38 in the meantime, enabling the `mvn install -Pdocker` command. Please note, however, that we're still in the process of streamlining the build process also for the AAS Registry & Submodel Registry. Thus, the command will for now only work for the repositories.

**Comment by FrankSchnicke** [comment]
> For example the concept description image doesn't have pagination, though if you build and run locally, you would have pagination information in response. This is very confusing.

I just double-checked, the SNAPSHOT version on DockerHub uses pagination. Did you have an older version in your local cache?

**Comment by mhrimaz** [comment]
You are right. The problem was with the image cache. pulling the image again solved the problem.

Thanks for your response. I will close the issue then.
